packages/f-checkout/src/components/tests/Address.test.js

```diff
@@ -0,0 +1,281 @@
+            i18n,
+            store: createStore(),
+            localVue,
+            propsData,
+            provide: () => ({
+                $v
+            })
+        });
+
+        // Assert
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    describe('computed ::', () => {
+        const isFieldEmptySpy = jest.spyOn(Address.methods, 'isFieldEmpty');
+        let wrapper;
+
+        describe('isAddressLine1Empty ::', () => {
+            it('should call `isFieldEmpty` with argument `line1`', async () => {
+                // Act
+                wrapper = shallowMount(Address, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    provide: () => ({
+                        $v
+                    })
+                });
+
+                // Assert
+                expect(isFieldEmptySpy).toHaveBeenCalledWith('line1');
+            });
+        });
+
+        describe('isAddressCityEmpty ::', () => {
+            it('should call `isFieldEmpty` with argument `city`', async () => {
+                // Arrange & Act
+                wrapper = shallowMount(Address, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    provide: () => ({
+                        $v
+                    })
+                });
+
+                // Assert
+                expect(isFieldEmptySpy).toHaveBeenCalledWith('city');
+            });
+        });
+
+        describe('isAddressPostcodeEmpty ::', () => {
+            it('should call `isFieldEmpty` with argument `postcode`', async () => {
+                // Arrange & Act
+                wrapper = shallowMount(Address, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    provide: () => ({
+                        $v
+                    })
+                });
+
+                // Assert
+                expect(isFieldEmptySpy).toHaveBeenCalledWith('postcode');
+            });
+        });
+
+        describe('isAddressPostcodeValid ::', () => {
+            it('should return `false` if postcode field has been touched, is not Valid and is not empty', async () => {
+                // Arrange && Act
+                $v.addressValidations.postcode.$dirty = true;
+                $v.addressValidations.postcode.isValidPostcode = false;
+
+                wrapper = shallowMount(Address, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    provide: () => ({
+                        $v
+                    }),
+                    computed: {
+                        isAddressPostcodeEmpty () {
+                            return false;
+                        }
```

Comment:
You can use an arrow function here to keep things concise. I think we were looking to use the shorthand format for computed properties in unit tests to keep them consistent.

```suggestion
                        isAddressPostcodeEmpty: () => false
```

Same for the tests below.